### DISCUSSION TITLE
Add tips?

### DISCUSSION THREAD

**Discussion by SuhailSumra** [benawad/dogehouse#2503] [Ideas]
Well how about a feature for tipping user via cryptocurrency? Like for example if in my feed i liked something and i want to encourage that user or just feel like being a good person and tip him with dogecoins or something? Since its "DOGEHOUSE" lets start with something like dogecoins that way we will be encouraging users and as well as maybe help in the increase of prices of dogecoins?

**Comment by dk-raw** [comment]
We will eventually implement that along with DogeNitro and DogeSubs.
